docs/bundler/vs-esbuild.md

```diff
@@ -333,6 +333,11 @@ In Bun's CLI, simple boolean flags like `--minify` do not accept an argument. Ot
 
 ---
 
+- `--minify-symbols`
```

Comment:
esbuild does not have --minify-symbols

Reply:
yeah, let's not have this as a flag. if `--minify-identifiers` is enabled, so should this be

Reply:
> yeah, let's not have this as a flag. if `--minify-identifiers` is enabled, so should this be

Do you want it entirely merged with identifiers so '{ minify: { identifiers: true } }' as well as '--minify-identifiers'?

I separated it out originally since it might be considered breaking for people who inspect symbols for any reason. Not a concern?